### PR TITLE
Versions lookup return latest branch hint

### PR DESCRIPTION
**Pull request by crisptrutski**
Refs jspm/npm#34

### CODE REVIEW

**Comment by guybedford** [comment]
If a package on github has `2.0.0-alpha` and `master` both of which are unstable, would the `install --edge` would select master?

File: github.js [hunk 1]
```diff
@@ -327,7 +327,7 @@ GithubLocation.prototype = {
           versions[version] = versionObj;
         }
 
-        resolve({ versions: versions });
+        resolve({ versions: versions, latest: "master" });
```
Comment (guybedford) [inline github.js]: To be super-pendatic, single quote?

**Comment by crisptrutski** [comment]
It would install a `1.x.y` - otherwise `master` if those don't exist.

**Comment by guybedford** [comment]
* `master` is an unstable latest
* `dev` can be the default edge if it exists

Do those conventions make sense?

**Comment by crisptrutski** [comment]
Sure. If `dev` exists perhaps `master` should be stable though.

File: github.js [hunk 2]
```diff
@@ -327,7 +327,7 @@ GithubLocation.prototype = {
           versions[version] = versionObj;
         }
 
-        resolve({ versions: versions });
+        resolve({ versions: versions, latest: 'master', edge: 'dev' });
```
Comment (guybedford) [inline github.js]: Perhaps only set these if they exist?

Comment (guybedford) [inline github.js]: I see you have the checks in the main code, which is great, deleted the comment.

**Pull request merged**